libraries/src/MVC/View/ListView.php

```diff
@@ -180,17 +180,12 @@ public function __construct(array $config)
      */
     public function display($tpl = null)
     {
-        // Prepare view data
-        $this->initializeView();
-
         $model = $this->getModel();
 
-        $errors = $model->getErrors();
-
-        // Check for errors.
-        if (!empty($errors)) {
-            throw new GenericDataException(implode("\n", $errors), 500);
-        }
+        $model->setUseExceptions(true);
+        
+        // Prepare view data
```

Comment:
```suggestion
        $model->setUseExceptions(true);

        // Prepare view data
```
Fix PHPCS whitespace found at end of line